recipe/meta.yaml

```diff
@@ -1,51 +1,37 @@
 {% set name = "qt-main" %}
-{% set version = "5.15.3" %}
+{% set version = "6.3.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  - url: https://download.qt.io/official_releases/qt/{{ version.rpartition('.')[0] }}/{{ version }}/single/qt-everywhere-opensource-src-{{ version }}.tar.xz
-    sha256: b7412734698a87f4a0ae20751bab32b1b07fdc351476ad8e35328dbe10efdedb
+  - url: https://download.qt.io/official_releases/qt/{{ version.rpartition('.')[0] }}/{{ version }}/single/qt-everywhere-src-{{ version }}.tar.xz
+    md5: 957a304773b281a4584f4c0254773456
```

Comment:
Is it possible to use sha256 here?

Reply:
qt only provides md5s, so its easier:
https://download.qt.io/official_releases/qt/6.3/6.3.1/single/